ccd-definition/CaseEventToFields/CareSupervision/ordersNeeded.json

```diff
@@ -14,12 +14,38 @@
     "LiveFrom": "01/01/2017",
     "CaseTypeID": "CARE_SUPERVISION_EPO",
     "CaseEventID": "ordersNeeded",
-    "CaseFieldID": "orders",
+    "CaseFieldID": "orderListToShow",
```

Comment:
I couldn't get it to work with representativeType, but one thing to note is that even if we were to use that field, we would have issues with the summary page because we wouldn't want to display representativeType again and this would lead to the issue where the summary page is blank. Currently to get around this, I do display the orderListToShow field but the label is blank, and I am using invisible characters for the elements so to the user all they see is a bit of whitespace